data/data/ovirt/main.tf

```diff
@@ -1,7 +1,10 @@
 provider "ovirt" {
-  url      = var.ovirt_url
-  username = var.ovirt_username
-  password = var.ovirt_password
+  url       = var.ovirt_url
+  username  = var.ovirt_username
+  password  = var.ovirt_password
+  cafile    = var.ovirt_cafile
```

Comment:
I'm pushing to removing this variable and have it in the bundle so I think you can remove it